apps/full-site-editing/full-site-editing-plugin/dotcom-fse/plugins/close-button-override/style.scss

```diff
@@ -1,6 +1,6 @@
-.post-type-wp_template_part .edit-post-fullscreen-mode-close__toolbar,
-.post-type-page .edit-post-fullscreen-mode-close__toolbar,
-.post-type-post .edit-post-fullscreen-mode-close__toolbar,
+.post-type-wp_template_part .edit-post-header .edit-post-fullscreen-mode-close,
```

Comment:
I think we should keep the old and new selectors here so that the button is still hidden for users on the old version

Reply:
👍 Fixed in b88c7e9900fc9e8dc5076ddb65b052d71d28a2cb